tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md

```diff
@@ -92,18 +92,45 @@ The expected failures in this file are from features in the owncloud/ocis repo.
+- [apiGraph/getUser.feature:144](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getUser.feature#L144)
+- [apiGraph/getUser.feature:145](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getUser.feature#L145)
+- [apiGraph/getUser.feature:146](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getUser.feature#L146)
+- [apiGraph/getUser.feature:147](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getUser.feature#L147)
+- [apiGraph/getUser.feature:148](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getUser.feature#L148)
+- [apiGraph/getUser.feature:149](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getUser.feature#L149)
+- [apiGraph/getUser.feature:150](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getUser.feature#L150)
+- [apiGraph/getUser.feature:151](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getUser.feature#L151)
+- [apiGraph/getUser.feature:152](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getUser.feature#L152)
+- [apiGraph/getUser.feature:153](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getUser.feature#L153)
+- [apiGraph/getUser.feature:154](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getUser.feature#L154)
 
 #### [Sharing to a group with an expiration date does not work #5442](https://github.com/owncloud/ocis/issues/5442)
 - [apiSpacesShares/shareSubItemOfSpace.feature:105](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpacesShares/shareSubItemOfSpace.feature#L105)
 
-#### [Space admin should not not be able to change the user quota](https://github.com/owncloud/ocis/issues/5475)
+#### [Space admin should not be able to change the user quota](https://github.com/owncloud/ocis/issues/5475)
 - [apiSpaces/spaceManagement.feature:149](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/spaceManagement.feature#L149)
 
 #### [Normal user can get expanded members information of a group](https://github.com/owncloud/ocis/issues/5604)
 - [apiGraph/getGroup.feature:100](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/getGroup.feature#L100)
 
+#### [Non-admin user tries to delete non-existing result 401](https://github.com/owncloud/ocis/issues/5738)
+- [apiGraph/deleteUser.feature:83](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/deleteUser.feature#L83)
+- [apiGraph/deleteUser.feature:84](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/deleteUser.feature#L84)
+- [apiGraph/deleteUser.feature:85](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/deleteUser.feature#L85)
```

Comment:
Not sure whether to expect this or not. IMO, non-admin user getting `401` while trying to delete non-existing user should be fine.
Also see https://github.com/owncloud/ocis/issues/5738#issuecomment-1471501646

CC @SwikritiT @SagarGi @grgprarup @PrajwolAmatya @individual-it

Reply:
I think its a bug, but likely low prio. See my comment in the issue